frontend/src/api/skills/getAllSkills.ts

```diff
@@ -0,0 +1,31 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import { AxiosError } from 'axios';
+import { useSnackbar } from 'notistack';
+
+import { apiClientWrapper } from 'api/base';
+import { API } from 'constants/routes';
+import { errorSnackbar } from 'constants/snackbarVariant';
+import { QUERY_KEYS } from 'constants/queryKeyConstants';
+import { SkillsList } from 'types/skill';
+
+const useGetAllSkills = (): UseQueryResult<SkillsList[], AxiosError> => {
+  const { enqueueSnackbar } = useSnackbar();
+  const handleSubmitError = (error: AxiosError) => {
+    enqueueSnackbar(error?.response?.data, errorSnackbar);
+  };
+  return useQuery(
+    [QUERY_KEYS.skills],
+    async () => {
+      const apiClient = apiClientWrapper();
+      const response = await apiClient.get(`${API.skills}`);
+      const skillsResponse: SkillsList = response.data;
+      return skillsResponse;
+    },
+    {
+      staleTime: 600000,
```

Comment:
I dont now, is it good solution? If we needed at updating this query data, we can do this inside mutation, that adding something new. Like queryCache.refetch. But if you think that this case is good - add time value to const